source/Calamari/Kubernetes/ManifestReporter.cs

```diff
@@ -55,7 +51,7 @@ public void ReportManifestApplied(string filePath)
             if (!FeatureToggle.KubernetesLiveObjectStatusFeatureToggle.IsEnabled(variables) && !OctopusFeatureToggles.KubernetesObjectManifestInspectionFeatureToggle.IsEnabled(variables))
                 return;
 
-            using (var yamlFile = fileSystem.OpenFile(filePath, FileAccess.ReadWrite))
+            using (var yamlFile = fileSystem.OpenFile(filePath, FileAccess.Read, FileShare.Read))
```

Comment:
@eddymoulton I've changed this to only require read access - should be ok yeah?

Reply:
Correct, that's a good change